rpm/pacemaker.spec.in

```diff
@@ -335,6 +335,7 @@ BuildRequires: %{pkgname_gettext} >= 0.18
 
 # Required for "make check"
 BuildRequires: libcmocka-devel >= 1.1.0
+BuildRequires: %{python_name}-psutil
```

Comment:
I think we also need python3-psutil installed on the CI builders before this will pass.

Reply:
https://github.com/kronosnet/ci-tools/pull/27

Reply:
spelling in this commit message (can use Build: rpm:)